framework/src/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -1267,6 +1263,28 @@ private boolean shouldSuppressWarnings(Tree tree, String err) {
     }
 
     /**
+     * Should unchecked code defaults be used for the kind of code indicated by the parameter
+     * @param kindOfCode source or bytecode
+     * @return whether unchecked code defaults should be used
+     */
+    public boolean useUncheckedCodeDefault(String kindOfCode) {
+        String option = this.getOption("useDefaultsForUncheckedCode", "-source,-bytecode");
+        if(option == null){
+            // if "useDefaultsForUncheckedCode" option is supplied without
+            // an argument, use the default.
+            option = "-source,-bytecode";
+        }
+
+        String[] args = option.split(",");
+        for (String arg : args) {
+            boolean value = !arg.contains("-");
+            if (arg.contains(kindOfCode)) {
```

Comment:
Not that I expect users to do this, but .contains is a bit fragile - I recommend doing an exact match after removing the "-" (if present). Otherwise "-sourcebytecode" would apply to both.